### ISSUE TITLE
Is there a definition of "dp" missing in the "rela_graph_node_update" class?

### ISSUE THREAD

**Issue by Sentence99** [zhanglabNKU/DMCF-DDI#1]
Is my understanding that this "dp" stands for dropout?

**Comment by zhanglabNKU** [comment]
Yes, here is the definition.
`self.dp = nn.Dropout(0.1)`

**Comment by Sentence99** [comment]
Thanks for your response!
